src/ape_accounts/_cli.py

```diff
@@ -57,8 +57,7 @@ def _list():
 @click.argument("alias")
 def generate(alias):
     if alias in accounts.aliases:
-        notify("ERROR", f"Account with alias '{alias}' already exists")
-        return
+        raise Abort(f"Account with alias '{alias}' is already in use")
```

Comment:
This is the bugfix, raising an Exception. It has the same effect, as far as CLI output goes, but the exist code is 1 instead of 0.

This is in `_import()` as well